colossalai/inference/core/engine.py

```diff
@@ -1,6 +1,9 @@
 from logging import Logger
 from typing import Optional
 
+from transformers import AutoConfig
+
+from ..config import ColossalInferConfig
```

Comment:
Why not put the config as part of the `core` module?

Reply:
Done